calebwagner/kneel-diamonds
# exported database copy to new module

I exported a copy of the database to a new module to make it the only module that directly interacts with the raw state.

Steps to check code:
1. git fetch --all
2. git checkout cjw-create-dataAccess.js
3. code .
4. make sure the database has been exported from database.js to dataAccess.js
5. check the otherOptions function which contains otherChoice array
